notebooks/stable-fast-3d/stable-fast-3d.ipynb

```diff
@@ -0,0 +1,1053 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #50.        Dinov2EmbeddingsPatched.interpolate_pos_encoding, model.image_tokenizer.model.embeddings
</pre><p> do you really need to create class for patching function. Does it not work if it is just function?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2388/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>